tools/TestFx/Live/InvokeLiveTestCITask.ps1

```diff
@@ -0,0 +1,29 @@
+[CmdletBinding(DefaultParameterSetName = "ByScriptFile")]
+param (
+    [Parameter(Mandatory)]
+    [bool] $UseWindowsPowerShell,
+
+    [Parameter(Mandatory, ParameterSetName = "ByScriptFile")]
+    [ValidateNotNullOrEmpty()]
+    [string] $ScriptFile,
+
+    [Parameter(Mandatory, ParameterSetName = "ByScriptBlock")]
+    [ValidateNotNullOrEmpty()]
+    [string] $ScriptBlock
+)
+
+if ($UseWindowsPowerShell) {
+    $process = "powershell"
+}
+else {
+    $process = "dotnet tool run pwsh"
+}
+
+switch ($PSCmdlet.ParameterSetName) {
+    "ByScriptFile" {
+        Invoke-Expression "$process -NoLogo -NoProfile -NonInteractive -File $ScriptFile"
+    }
+    "ByScriptBlock" {
+        Invoke-Expression "$process -NoLogo -NoProfile -NonInteractive -Command $ScriptBlock"
```

Comment:
I think need the quote for `$ScriptBlock` here.

Reply:
$ScriptFile needs the quotes as the path may have space. Quotes for $ScriptBlock seems not necessary. Do you have samples?

Reply:
This will get wrong result if the ScriptBlock string doesn't contains big parantheses. So I suggest to add a pair of big parantheses to prevent input error.